invite/src/invite_bot.ts

```diff
@@ -57,7 +73,17 @@ export class InviteBot {
    * Parses a comment for invitation macros.
    */
   parseMacros(comment: string): Record<string, InviteAction> {
-    return {};
+    const macros: Record<string, InviteAction> = {};
+    const matches = comment.match(FULL_MACRO_REGEX);
+
+    if (matches) {
+      matches.forEach((macro: string) => {
+        const [macroString, username] = macro.substr(1).split(' @');
+        macros[username] = INVITE_MACROS[macroString.toLowerCase()];
```

Comment:
Isn't macroString here going to be '/invite' instead of 'invite'?

Reply:
`.substr(1)`

Reply:
lol